### ISSUE TITLE
delta calculation reports Error

### ISSUE THREAD

**Issue by qiquanbuaa** [YuMan-Tam/deep-hedging-demo#3]
Dear YuMan,
Thanks a lot for your help,and I have another issue in coding error,
When I ran this line: 
nn_delta = submodel(I_range)
I got InvalidArgumentError: Matrix size-incompatible: In[0]: [1,101], In[1]: [1,15] [Op:MatMul]

![image](https://user-images.githubusercontent.com/43933149/80865154-ef4f2000-8cb9-11ea-817c-d6742d96a3c4.png)

It seems that the the defined Delta_submodel have matrix size-incompatible problems.
But the Delta_submodel doesn't seem to specify the dimension of the input. Could you seek out where the bug is?
Thanks a lot for your help! I will appreciate it!

**Comment by YuMan-Tam** [comment]
I run the latest version and there doesn't seem to be any issue. May be try the newest version? It might simply that I messed up the updates.

The variable ```I_range``` is supposed to be any ``1D-np.arrays``. Basically, the optimal strategy is a function of the underlying stock price. Therefore, for each element in the array, the trained network should produce a number.

One thing: This is intended to be a personal project so do not expect any sustainable support. This repository is really unstable because I might constantly mess with the ```master``` branch, which might results in bugs appears here and there. Use it at your own risk.

There will probably be major changes to the code base anyway if I got time to expand this project.
